weather-app/src/actions/getWeather.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { useParams } from 'react-router-dom'
+import axios from 'axios'
+import API_KEY from '../secrets'
+
+
+
+const GetWeather = () => {
+    // const [ showCity, setShowCity ] = useState([]);
+    const [ showWeather, setShowWeather ] = useState('');
```

Comment:
You can have a lot of these states in one object so you only have to keep track of one thing